packages/react/lib/template.js

```diff
@@ -1,9 +1,19 @@
 const esc = require('serialize-javascript');
 
-const variable = ({ name, value }) => `<script>${name}=${esc(value)}</script>`;
 const cssLink = (css) => `<link rel="stylesheet" href="/${css}" />`;
 const jsLink = (js) => `<script src="/${js}"></script>`;
 
+const serializeGlobals = (globals) => {
+  var entries = Object.entries(globals);
```

Comment:
`var` could be `const` or `let`. Afaik no need to resort to es5 here.

Reply:
Oops, fixed.